popup.js

```diff
@@ -1,38 +1,81 @@
-let active = document.getElementById('active');
+const active = document.getElementById('active');
+const activeLabel = document.getElementById('on-switch-label');
+const intervalSlider = document.getElementById('reload-int-input');
+const intervalDisplay = document.getElementById('reload-int-display');
+const settingsContainer = document.getElementById('settings-container');
+const seconds = document.getElementById('sec');
+const minutes = document.getElementById('min');
 
+// Maintain checked input box if this window has active reload
 chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
   let tab = tabs[0];
   chrome.storage.local.get(tab.id.toString(), (items) => {
-      if(items[tab.id]){
-          console.log("found!");
-          active.checked = true;
-          
+    // Load previous user input settings
+    if (items[tab.id]) {
+      // Load unit check box
+      if (items[tab.id].unit === 'min') {
+        minutes.checked = true;
+      } else {
+        seconds.checked = true;
       }
+
+      // Load Interval display
+      intervalDisplay.innerHTML = items[tab.id].input;
+      intervalSlider.value = items[tab.id].input;
+    }
+
+    // Check off checkbox if currently reloading
+    if(items[tab.id] && items[tab.id].active){
+        active.checked = true;
```

Comment:
Looks like you missed this here.
```suggestion
        active.checked = true;
        activeLabel.innerHTML = 'ON';
```

Reply:
> Thanks for the compliment! I intentially disabled the settings GUI except for the on/off switch when it's running. I wasn't sure what kind of consequence it would have with the setInterval() if changes were keep getting made to the storage more than once before it stopped running (since clearInterval only occurs when the switch is turned off), so this was an unsophisticated solution to prevent setInterval stacking.

If we are disabling, please mention in a dynamic error toast that it is disabled while in reload state already.